### PR TITLE
Add advanced ventilation options.

### PR DESCRIPTION
**Pull request by beshaya**
All 1/4:
* HEPA filters
* Planes/Trains
* Partially Enclosed

Closes: #492

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -147,6 +147,10 @@
     "repeated": "Household member [{{ percentage }} chance of transmission per week]",
     "partner": "Partner / spouse [{{ percentage }} chance of transmission per week]",
     "indoor": "Indoor",
+    "filtered": "Indoors with a HEPA filter (flow rate 5x room size)",
+    "transit": "An airplane or train with air filtration",
+    "carWindowsDown": "A moving car with the windows rolled down",
```
Comment (blanchardjeremy) [inline src/locales/en.json:152]: I noticed you used camel casing here. We're using underscores elsewhere.  It'd be good to keep it consistent. I don't care which one we use.

File: src/locales/en.json [hunk 2]
```diff
@@ -147,6 +147,10 @@
     "repeated": "Household member [{{ percentage }} chance of transmission per week]",
     "partner": "Partner / spouse [{{ percentage }} chance of transmission per week]",
     "indoor": "Indoor",
+    "filtered": "Indoors with a HEPA filter (flow rate 5x room size)",
```
Comment (blanchardjeremy) [inline src/locales/en.json:150]: I wonder how we can help people make sense of this. Maybe here by adding an info alert below the input box if they select this option. Or we could add it to the whitepaper and link them to it. 

In either case, I'd love to be able to give them some basic instructions. Even something like "Google the flow rate of your HEPA filter. Make sure it's rated XYZ or above. Multiply the with x length x height of your room. If the flow rate is greater than 5x that room size per hour, then you're good to go." (Except I'm butchering the explanation, but you get the idea.)

Whatcha think about adding something like that.

**Review by blanchardjeremy — CHANGES_REQUESTED** [review]
Looks great overall. Thanks for doing this!! Couple of comments.

File: src/locales/en.json (hunk 2 above)
Comment (beshaya) [reply src/locales/en.json:150]: Yeah, I think a link to the whitepaper is better here.

File: src/locales/en.json (hunk 1 above)
Comment (beshaya) [reply src/locales/en.json:152]: Fixed

File: src/locales/en.json (hunk 2 above)
Comment (beshaya) [reply src/locales/en.json:150]: Done, ptal.

File: src/components/calculator/ActivityRiskControls.tsx [hunk 3]
```diff
@@ -51,6 +51,15 @@ export const ActivityRiskControls: React.FunctionComponent<{
           <Trans>calculator.precautions.no_intimate_bonus_outdoors</Trans>
         </div>
       ) : null}
+      {data.setting === 'filtered' && (
+        <div className="warning">
+          <Trans>calculator.precautions.filtered_ensure_airflow</Trans>{' '}
+          <Link to="/paper/14-research-sources#ventilation">
+            <Trans>calculator.precautions.filtered_ensure_airflow_link</Trans>
```
Comment (blanchardjeremy) [inline src/components/calculator/ActivityRiskControls.tsx:58]: I've been learning way more about translation than I ever knew. One pro tip: don't assume grammer structure. Some languages might not end on the word "here". Instead, you can put the link inside a single <Trans> element. Then in the en.json file you can follow the pattern you see elsewhere of using <2>Foo</2> to fill in the link tag.

Let me know if you have any questions.

File: src/posts/paper/14-research-sources.ts [hunk 4]
```diff
@@ -77,6 +77,21 @@ The sources above hint at up to a 100x benefit. Fellow armchair modeler Peter Hu
...
+* For a normal room height (8ft), ensure \`CFM > 2/3 * length * width\`
+
+The derivation of this rule is as follows:
+\`\`\`
+Refreshes/hr = 60 min/hr * CFM ft^3/min / (length ft * width ft * height ft) = 5
+60 * CFM / (length * width * height) = 5
+CFM = 5 * (length * width * height) / 60
+CFM = 1/12 * (length * width * height)
+if height = 8ft,
+CFM = 2/3 * (square footage)
```
Comment (blanchardjeremy) [inline src/posts/paper/14-research-sources.ts:92]: Thank you! Much more clear to users.

**Review by blanchardjeremy — CHANGES_REQUESTED** [review]
One last change needed.

File: src/components/calculator/ActivityRiskControls.tsx (hunk 3 above)
Comment (beshaya) [reply src/components/calculator/ActivityRiskControls.tsx]: Done! That's a really cool feature.

**Review by blanchardjeremy — APPROVED** [review]
Looks great.

**Pull request merged**